Jenkinsfile

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env groovy
+
+REPOSITORY = 'rummager'
+
+node {
+  def govuk = load '/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy'
+
+  properties([
+    buildDiscarder(
+      logRotator(
+        numToKeepStr: '50')
+      ),
+    [$class: 'RebuildSettings', autoRebuild: false, rebuildDisabled: false],
+    [$class: 'ThrottleJobProperty',
+      categories: [],
+      limitOneJobWithMatchingParams: true,
+      maxConcurrentPerNode: 1,
+      maxConcurrentTotal: 0,
+      paramsToUseForLimit: 'Rummager',
+      throttleEnabled: true,
+      throttleOption: 'category'],
+  ])
+
+  try {
+    stage("Checkout") {
+      checkout scm
+    }
+
+    stage("git merge") {
+      govuk.mergeMasterBranch()
+    }
+
+    stage("bundle install") {
+      govuk.bundleApp()
+    }
+
+    stage("Delete static symlinks and reconnect") {
+      sh '''for d in images javascripts templates stylesheets; do
+              rm -f public/$d
+              ln -s ../../../Static/workspace/public/$d public/
+            done
+         '''
+    }
+
+    stage("rubylinter") {
+      govuk.rubyLinter()
+    }
+
+    stage("Run tests") {
+      govuk.setEnvar('USE_SIMPLECOV', 'true')
+      govuk.setEnvar('RACK_ENV', 'test')
+      govuk.runRakeTask('ci:setup:minitest test --trace')
+    }
+
+    // Commented out for testing
```

Comment:
If it's going to stay commented out I think you should update this comment. It sounds like we're disabling this until we switch over to the new system, is that right?